src/dmd/lexer.d

```diff
@@ -429,7 +429,7 @@ class Lexer
                     goto case_ident;
                 p++;
                 t.value = hexStringConstant(t);
-                deprecation("Built-in hex string literals are deprecated, use `std.conv.hexString` instead.");
+                error("Built-in hex string literals are deprecated, use `std.conv.hexString` instead.");
```

Comment:
Maybe "no longer supported" instead of deprecated?

Reply:
Perhaps a bit too late, but can we include the literal in the suggested fix? Basically exactly how the call to `hexString` need to look like.

Reply:
You're killing me Jacob, but yes that should be possible.  I'll try.

Reply:
Sorry Jacob, I tried a little but I'm getting other problems.

```
fail_compilation/lexer1.d(30): Error: 
Program received signal SIGSEGV, Segmentation fault.
```
I'm not sure how to extract the string out of the `Token` type.  If you have some wisdom, pass it on and I'll give it another try.

Reply:
What does `toChars` give you? But otherwise don't bother.

Reply:
I think I figured it out.

Reply:
Maybe the text error would be more accurate if it said "built-in hex string literals are obsolete".